### ISSUE TITLE
Colour contrast fail for <nav> items in header on focus

### ISSUE THREAD

**Issue by shabana-ali** [hmrc/accessibility#36]
![screenshot-signoutlink-contrastfail](https://user-images.githubusercontent.com/59563139/87170203-efc2f300-c2c8-11ea-823b-097f05752374.png)

Workaround fix:

```css
.hmrc-header.hmrc-header--with-additional-navigation .hmrc-sign-out-nav__link:focus {
    color: #0B0C0C;
}
```
